src/Payments.sol

```diff
@@ -488,10 +487,12 @@ contract Payments is
         uint256 lockupFixed
     ) internal {
         Account storage payer = accounts[rail.token][rail.from];
+        bool isIncrease = period > rail.lockupPeriod ||
+            lockupFixed > rail.lockupFixed;
 
         // Settle account lockup as much as possible
         // Only require full settlement if increasing period or fixed lockup
-        if (period > rail.lockupPeriod || lockupFixed > rail.lockupFixed) {
+        if (isIncrease) {
```

Comment:
@Stebalien We probably do not need the check on line 500/501 below but keeping it does make it easy for me to reason about things. Any strong opinions ?